http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala

```diff
@@ -403,7 +403,7 @@ class RuntimeServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
 
       _ <- if (hasStopPermission) F.unit else F.raiseError[Unit](ForbiddenError(userInfo.userEmail))
       // throw 409 if the cluster is not stoppable
-      _ <- if (runtime.status.isStoppable) F.unit
+      _ <- if (runtime.status.isStoppable || runtime.status.isStopping) F.unit
```

Comment:
It looks like if `status` is one of `stoppingStatuses`, this would go on to publish a `StopRuntimeMessage`. I'd think what we want is then to neither throw an error nor take any actions since there's no need to attempt to stop a runtime that is/will be stopped. Am I missing something?

Reply:
oh good point..maybe we should take out `PreStopping, Stopping`..It shouldn't hurt to republish a `StopRuntime` message, but if for some reason back leo drops the message, republishing the message will fix it

Reply:
Not sure if I understand completely. 

1. To clarify the desired behaviour…when a user attempts to stop compute on the UI that is stopped/stopping/prestopping, we want Leo to return success, no?
2. If we have a problem of a back Leo dropping PubSub messages, wouldn't it warrant a more holistic solution than a one-off solution for a very particular case?

Reply:
hmm okay I think I understand what you're saying. I've moved the check to wrap the enqueue at the end so if it's already stopping it won't enqueue another message - testing it out now but is that more what you're thinking?

Reply:
> hmm okay I think I understand what you're saying. I've moved the check to wrap the enqueue at the end so if it's already stopping it won't enqueue another message - testing it out now but is that more what you're thinking?

exactly :)

> Not sure what more holistic solution you're thinking..But the point is more of in back leo, if it gets a second
StopRuntime message, it should be able to deal with it gracefully

I was trying to say that I don't think we should publish a message redundantly.